src/mjolnir/valhalla_ways_to_edges.cc

```diff
@@ -44,15 +45,17 @@ bool ParseArguments(int argc, char* argv[]) {
       "\n"
       " Usage: ways_to_edges [options]\n"
       "\n"
-      "ways_to_edges is a program that creates a list of edges for each OSM way "
-      "on the local level tiles."
+      "ways_to_edges is a program that creates a list of edges for each OSM way. "
+      "Configure the levels with min_level."
       "\n"
       "\n");
 
   options.add_options()("help,h", "Print this help message.")("version,v",
                                                               "Print the version of this software.")(
       "config,c", boost::program_options::value<filesystem::path>(&config_file_path)->required(),
-      "Path to the json configuration file.")
+      "Path to the json configuration file.")(
+      "min_level", boost::program_options::value<uint16_t>(&min_level),
+      "Min level to export: 0 will export all levels, 1 will export all levels but 0 and so on.")
```

Comment:
Its a bit weird because we say min level but the smaller levels have larger numbers. Honestly i wouldnt be against just changing this thing to default to going through all the levels and just ignoring the break in compatibility but if you think the flexibility is needed maybe we can just parse comma separate level numbers as a string and then validate them as you do below after turning them into an unordered_set of uint8_t. there is a snippet for doing soemthign like this in prime_server that you could re-use if it sounds interesting to you: https://github.com/kevinkreiser/prime_server/blob/master/src/prime_helpers.hpp#L7-L25

if not not worries just a suggestion to reduce confusion

Reply:
Ah no, totally fine for _me_ to drop that sort of flexibility/backwards compatibility.. Maybe someone else has some strong feeling about this? Otherwise I'll drop it..

Reply:
Seems no one cares too much, let's do all levels by default then.